src/test/java/lotto/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package lotto;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.ParameterizedTest.DEFAULT_DISPLAY_NAME;
+
+class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set - size 메서드 테스트")
+    @Test
+    void size() {
+        assertThat(numbers).hasSize(3);
+    }
+
+    @ParameterizedTest(name = "Set - contains 테스트 (Parameterized value) " + DEFAULT_DISPLAY_NAME)
+    @ValueSource(ints = { 1, 2, 3 })
+    void contains(int input) {
+        assertThat(numbers.contains(input)).isTrue();
+    }
+
+    @ParameterizedTest(name = "Set - contains 테스트 (Parameterized csv) " + DEFAULT_DISPLAY_NAME)
+    @CsvSource(value = { "1:true", "2:true", "3:true", "4:false", "5:false" }, delimiter = ':')
+    void containsTrueOrFalse(String inputString, String expectedString) {
+        //given:
+        int input = Integer.parseInt(inputString);
+        boolean expected = Boolean.parseBoolean(expectedString);
+        //then:
+        assertThat(numbers.contains(input)).isEqualTo(expected);
+    }
```

Comment:
원시 타입으로 파싱하는 부분은 jupiter 에서 제공해주기 때문에
특별한 이유가 아니라면 파라미터 타입으로 바로 원시 타입으로 받으면 깔끔해질 것 같아요!!

```suggestion
    @ParameterizedTest(name = "Set - contains 테스트 (Parameterized csv) " + DEFAULT_DISPLAY_NAME)
    @CsvSource(value = { "1:true", "2:true", "3:true", "4:false", "5:false" }, delimiter = ':')
    void containsTrueOrFalse(int input, boolean expected) {
        assertThat(numbers.contains(input)).isEqualTo(expected);
    }
```

Reply:
넵! 확인했습니다. 데이터 파싱하는 api는 다양한 타입이 지원이 될 텐데, String으로 받아서 사용할 것이라고 혼자 추측 해 버렸네요 ㅎㅎ
앞으로의 api 사용에 조금 더 주의를 기울여야 겠습니다. 코멘트 감사합니다! ☺️